scripts/tests/test_zkai_minimal_transformer_block_benchmark_gate.py

```
@@ -0,0 +1,149 @@
+    def test_rejects_statement_binding_removal(self) -> None:
+        payload = gate.build_payload()
+        payload["benchmark_spec"]["public_statement_bindings"] = []
+        payload["payload_commitment"] = gate.payload_commitment(payload)
+        with self.assertRaises(gate.MinimalBlockBenchmarkError):
+            gate.validate_payload(payload)
+
+    def test_rejects_source_digest_drift(self) -> None:
+        payload = gate.build_payload()
+        payload["source_artifacts"][0]["file_sha256"] = "0" * 64
+        payload["payload_commitment"] = gate.payload_commitment(payload)
+        with self.assertRaises(gate.MinimalBlockBenchmarkError):
```

Ruff (0.15.12)

[warning] Use `pytest.raises` instead of unittest-style `assertRaises`

Replace `assertRaises` with `pytest.raises`

(PT027)